src/main/twirl/views/sectionForm.scala.html

```diff
@@ -19,12 +22,15 @@
 
     <div class="grid-row">
         <div class="column-two-thirds">
-            <p>
-                <a href="@controllers.routes.ApplicationController.show(app.id)">&lt; Return to application overview</a>
-            </p>
-            <h2>@formatId(opportunity.id.id): @opportunity.title</h2>
-
-            <h1>@formSection.title</h1>
+            <div class="breadcrumbs">
+                <ol style="margin: 0;padding:0;">
```

Comment:
I think we should consider moving this style code to a class instead.  What do you think?